scripts/packaging/createnugetpackages.ps1

```diff
@@ -75,13 +75,34 @@ try
             Write-Error "Building InEditor WindowsStandalone32 Failed! See log file for more information $(Get-Location)\Logs\Build.InEditor.WindowsStandalone32.$($Version).log";
         exit($lastexitcode)
     }
+    Write-Output "============ Building InEditor WSA ============ "
```

Comment:
The order in which the targets are built is different than in the CI template... which might be fine but I thought it's worth to point that out.

Reply:
@andreiborodin suggested this ordering. I will work with him to ensure the CI template is appropriately updated.